scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -442,6 +442,10 @@ class FossId internal constructor(
         projectCode: String,
         projectName: String
     ): String {
+        if ("@" in url) {
```

Comment:
Would it make sense to add an issue to [ScanSummary](https://github.com/oss-review-toolkit/ort/blob/9ce597968e1fb270e0aa8c68f7b16b5f632d6606/model/src/main/kotlin/ScanSummary.kt#L75) in this case?

Reply:
This would make sense.
However, this would complicate the implementation because we are in a function deep in the call tree, and it would be cumbersome to bubble the issues up the tree.
If it was an error case,  we could short circuit the logic with an exception.